drupal/tpl/drupal-rc.yaml

```diff
@@ -0,0 +1,66 @@
+#helm:generate helm tpl -d tpl/values.toml -o manifests/drupal-rc.yaml $HELM_GENERATE_FILE
```

Comment:
this line will change to

`#helm:generate helmc tpl -d tpl/values.toml -o manifests/drupal-rc.yaml $HELM_GENERATE_FILE`